### PR TITLE
Adding a Blockchain Import Guide 

### PR DESCRIPTION
**Pull request by KeeJef**
I'm new at using Github so just tell me if i got anything catastrophically wrong  

This should just be a simple guide on how to download the blockchain or import it from and outside source

### CODE REVIEW

File: resources/user-guides/index.md [hunk 1]
```diff
@@ -1,77 +1,78 @@
...
+                    <div class="col">
+                        <h2>Wallets</h2>
+                    </div>
+                </div>
+<div class="row start-xs" markdown="1">
+
+[Getting started with the CLI wallet](monero-wallet-cli.html)
+[How to make a view-only wallet](view_only.html)
+[How to prove payment](prove-payment.html)
+[Importing the Monero Blockchain from an outside source] (Importing_Blockchain.html)
```
Comment (MaxXor) [inline resources/user-guides/index.md]: You have to remove the space between `]` and `(` to make the link working.

Comment (KeeJef) [inline resources/user-guides/index.md]: Ok i think I've committed that now

Comment (MaxXor) [inline resources/user-guides/index.md]: Yes 👍

File: resources/user-guides/Importing_Blockchain.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+layout: static_page
+title: Importing the Monero Blockchain from an outside source 
+---
+
+# Importing the Blockchain to Monero GUI wallet (Windows)
+
+### Step 1
+
+Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw, you can skip this step if you are importing the Blockchain from another source.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: There should be a '.' or ';' after blockchain.raw. This is a run on sentence.

File: resources/user-guides/Importing_Blockchain.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+# Importing the Blockchain to Monero GUI wallet (Windows)
+
+### Step 1
+
+Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw, you can skip this step if you are importing the Blockchain from another source.
+
+### Step 2
+
+Find the Path of your Monero wallet (the folder where you extracted your wallet) for example mine is.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: > Find the Path of your Monero wallet (the folder where you extracted your wallet) for example mine is.

Period after ')'. This is a run on sentence.
Does Path need to be capitalized?

File: resources/user-guides/Importing_Blockchain.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+`D:\monero-gui-0.10.3.1-beta2`
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: Latest release is monero-gui-0.10.3.1.zip (no beta2). Default zip action on Windows is to extract to folder named the same as zip. For consistency, possibly change this to match the most recent release's folder name that they are likely to see?

File: resources/user-guides/Importing_Blockchain.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+Yours might be different depending on where you decided to download your wallet and what version of the Monero wallet you have.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: > Yours might
Consider changing to "Your path may" for clarity.

File: resources/user-guides/Importing_Blockchain.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+### Step 4
+
+Open a Command Prompt window. You can do this by pressing the Windows key + R, and then typing in the popup box `CMD`
+
+### Step 5
+
+Now you need to navigate using the CMD window to the path of your Monero wallet. You can do this by typing. 
+
+`cd THE PATH OF YOUR MONERO WALLET HERE`
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: > THE PATH OF YOUR MONERO WALLET HERE

Consider changing this to something that makes it obvious you're talking about a file path and use it throughout the document. i.e. C:\PATH\OF\YOUR\MONERO\WALLET.

File: resources/user-guides/Importing_Blockchain.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+Now you need to navigate using the CMD window to the path of your Monero wallet. You can do this by typing. 
+
+`cd THE PATH OF YOUR MONERO WALLET HERE` 
+
+It should look something like.
+
+`cd D:\monero-gui-0.10.3.1-beta2`
+
+If for some reason your Monero wallet is on another drive you can use `DriveLetter:` for example if your Monero wallet was on your D drive then before using the cd command you would do `D:`
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: > If for some reason your Monero wallet is on another drive you can use
Consider dropping 'some reason' (non-technical language/adds nothing).

File: resources/user-guides/Importing_Blockchain.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+### Step 6
+
+Now type in your command prompt window.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: Consider changing the '.' to a ':'.

File: resources/user-guides/Importing_Blockchain.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+`monero-blockchain-import --verify 1 --input-file YOUR BLOCKCHAIN FILE PATH HERE`
+
+For example I would type.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: Consider changing the '.' to a ':'.

File: resources/user-guides/Importing_Blockchain.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+`monero-blockchain-import --verify 1 --input-file C:\Users\KeeJef\Downloads\blockchain.raw`
+
+If you downloaded the Blockchain from a trusted, reputable source you may set `verify 0` this will reduce the amount of time to sync the Blockchain.  
+
+### Step 7
+
+After the the Blockchain has finished syncing up you can open your monero wallet normally. Your downloaded blockchain.raw can be deleted.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: > monero
Capitalize brand name.

File: resources/user-guides/Importing_Blockchain.md [hunk 11]
```diff
@@ -7,15 +7,15 @@ title: Importing the Monero Blockchain from an outside source
 
 ### Step 1
 
-Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw, you can skip this step if you are importing the Blockchain from another source.
+Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw ; you can skip this step if you are importing the Blockchain from another source.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: Punctuation whitespace issue.

File: resources/user-guides/Importing_Blockchain.md [hunk 12]
```diff
@@ -7,15 +7,15 @@ title: Importing the Monero Blockchain from an outside source
 
 ### Step 1
 
-Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw, you can skip this step if you are importing the Blockchain from another source.
+Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw ; you can skip this step if you are importing the Blockchain from another source.
 
 ### Step 2
 
-Find the Path of your Monero wallet (the folder where you extracted your wallet) for example mine is.
+Find the path of your Monero wallet (the folder where you extracted your wallet). for example mine is.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: > for example mine is.

Capitalize "for".
Change the period to a colon.

File: resources/user-guides/Importing_Blockchain.md [hunk 13]
```diff
@@ -33,29 +33,29 @@ Open a Command Prompt window. You can do this by pressing the Windows key + R, a
 
 Now you need to navigate using the CMD window to the path of your Monero wallet. You can do this by typing. 
 
-`cd THE PATH OF YOUR MONERO WALLET HERE` 
+`cd C:\YOUR\MONERO\WALLET\FILE\PATH\HERE` 
 
 It should look something like.
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: Colon instead of period.

File: resources/user-guides/Importing_Blockchain.md [hunk 14]
```diff
@@ -33,29 +33,29 @@ Open a Command Prompt window. You can do this by pressing the Windows key + R, a
...
 ### Step 6
 
-Now type in your command prompt window.
+Now type in your command prompt window :
 
-`monero-blockchain-import --verify 1 --input-file YOUR BLOCKCHAIN FILE PATH HERE`
+`monero-blockchain-import --verify 1 --input-file C:\YOUR\BLOCKCHAIN\FILE\PATH\HERE`
 
-For example I would type.
+For example I would type :
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: Punctuation whitespace issue.

File: resources/user-guides/Importing_Blockchain.md [hunk 15]
```diff
@@ -33,29 +33,29 @@ Open a Command Prompt window. You can do this by pressing the Windows key + R, a
...
-`cd D:\monero-gui-0.10.3.1-beta2`
+`cd D:\monero-gui-0.10.3.1`
 
-If for some reason your Monero wallet is on another drive you can use `DriveLetter:` for example if your Monero wallet was on your D drive then before using the cd command you would do `D:`
+If your Monero wallet is on another drive you can use `DriveLetter:` for example if your Monero wallet was on your D drive then before using the cd command you would do `D:`
 
 ### Step 6
 
-Now type in your command prompt window.
+Now type in your command prompt window :
```
Comment (QuickBASIC) [inline resources/user-guides/Importing_Blockchain.md]: Punctuation whitespace issue.

**Comment by fluffypony** [comment]
Thanks for your contribution, @KeeJef! This needs to be rebased to fix a conflict - are you able to do that?

**Comment by KeeJef** [comment]
Yeah i've just realized after deleting the whole index.md file i have no fucking idea what a rebase is, Github terms are overwhelming. can someone help me here

**Pull request closed without merging**